Kickstarter-iOS/Views/Controllers/ManagePledgePaymentMethodView.swift

```diff
@@ -0,0 +1,147 @@
+import KsApi
+import Library
+import Prelude
+import UIKit
+
+final class ManagePledgePaymentMethodView: UIView {
+  // MARK: - Properties
+
+  private lazy var adaptableStackView: UIStackView = { UIStackView(frame: .zero) }()
+  private lazy var cardImageView: UIImageView = { UIImageView(frame: .zero) }()
+  private lazy var cardLabelsStackView: UIStackView = { UIStackView(frame: .zero) }()
+  private lazy var expirationDateLabel: UILabel = { UILabel(frame: .zero) }()
+  private lazy var lastFourLabel: UILabel = { UILabel(frame: .zero) }()
+  private lazy var rootStackView: UIStackView = { UIStackView(frame: .zero) }()
+  private lazy var titleLabel: UILabel = { UILabel(frame: .zero) }()
+
+  private let viewModel = CreditCardCellViewModel()
+
+  // MARK: - Lifecycle
+
+  override init(frame: CGRect) {
+    super.init(frame: frame)
+
+    self.configureViews()
+    self.setupConstraints()
+    self.bindViewModel()
+  }
+
+  required init?(coder _: NSCoder) {
+    fatalError("init(coder:) has not been implemented")
+  }
+
+  // MARK: - Configuration
+
+  public func configure(with card: GraphUserCreditCard.CreditCard) {
+    self.viewModel.inputs.configureWith(creditCard: card)
+  }
+
+  private func configureViews() {
+    _ = self
+      |> \.accessibilityElements .~ self.subviews
+
+    _ = ([self.lastFourLabel, self.expirationDateLabel], self.cardLabelsStackView)
+      |> ksr_addArrangedSubviewsToStackView()
+
+    _ = ([self.cardImageView, self.cardLabelsStackView], self.adaptableStackView)
+      |> ksr_addArrangedSubviewsToStackView()
+
+    _ = ([self.titleLabel, self.adaptableStackView], self.rootStackView)
+      |> ksr_addArrangedSubviewsToStackView()
+
+    _ = (self.rootStackView, self)
+      |> ksr_addSubviewToParent()
+      |> ksr_constrainViewToEdgesInParent()
+  }
+
+  // MARK: - Styles
+
+  override func bindStyles() {
+    super.bindStyles()
+
+    _ = self.adaptableStackView
+      |> checkoutAdaptableStackViewStyle(
+        self.traitCollection.preferredContentSizeCategory.isAccessibilityCategory
+      )
+      |> adaptableStackViewStyle
+
+    _ = self.cardImageView
+      |> cardImageViewStyle
+
+    _ = self.cardLabelsStackView
+      |> cardLabelsStackViewStyle
+
+    _ = self.expirationDateLabel
+      |> cardExpirationDateLabelStyle
+
+    _ = self.lastFourLabel
+      |> cardLastFourLabelStyle
+
+    _ = self.rootStackView
+      |> checkoutCardStackViewStyle
+
+    _ = self.titleLabel
+      |> titleLabelStyle
+  }
+
+  // MARK: - View model
+
+  override func bindViewModel() {
+    super.bindViewModel()
+
+    self.expirationDateLabel.rac.text = self.viewModel.outputs.expirationDateText
+    self.lastFourLabel.rac.text = self.viewModel.outputs.cardNumberTextShortStyle
+    self.viewModel.outputs.cardImage
```

Comment:
Can we add empty line here to separate `rac` and `output` signals?

Reply:
On a second thought, would it be worth it adding reactive extension on the `UIImageView` to being able to bind `image` property? 🤔

Reply:
That's a good idea. I can create a ticket for that to be done in a different PR.